catalog/migrations/0006_alter_bookinstance_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.4 on 2021-06-23 13:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('catalog', '0005_bookinstance_borrower'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='bookinstance',
+            options={'ordering': ['due_back'], 'permissions': (('can_mark_returned', 'Set book as returned'),)},
+        ),
+    ]
```

Comment:
You could have used single migration file to make changes in the db. It looks a little odd to see two migration files in a single PR.

Reply:
I have updated and included a single migration file now